docs/user/openstack/provider_networks.md

```diff
@@ -0,0 +1,114 @@
+  - [Deploying cluster with primary interface on a provider network with IPI](#deploying-cluster-with-primary-interface-on-a-provider-network-with-ipi)
+  - [Known issues](#known-issues)
+
+
+## Introduction
+
+Provider networks map directly to an existing physical network in a data center.
+Example of network types include flat (untagged), VLAN (802.1Q tagged) and VXLAN. 
+OpenShift clusters that are installed on provider networks do not require tenant networks or floating IP addresses (FIPs).
+Therefore, the OpenShift installer does not create these resources during installation.
+More information can be found about provider networks terminology [here][1].
+
+Here is a basic architecture of one OCP cluster running on a provider network and another one
+on a tenant network:
+
+![OCP on a provider network](provider-network.png)
+
+
+## Prerequisites
+
+* The [Neutron service][2] is enabled and accessible through the [OpenStack Networking API][3].
+* The Neutroni service is configured with the [port-security and allowed-address-pairs][4] extensions so the installer can
+  add the `allowed_address_pairs` attribute to ports.
+
+
+## Considerations when creating provider networks
+
+* The project that you use to install OpenShift must own the provider network and subnet.
+
+    The provider network and subnet must be owned by the project that is used to install OpenShift instead of `admin`.
+    If they are not, Terraform fails while creating ports. 
+
+    It is important that the provider network and subnet are owned by the same project that will be used
+    to install OpenShift (from the clouds.yaml) and we don't want them to be owned by `admin` otherwise
+    it'll cause Terraform to fail creating the ports.
+
+    Example where our project name is `openshift`:
+
+        openstack network create --project openshift (...)
+        openstack subnet create --project openshift (...)
+
+    More information can be found about how to create provider networks [here][5].
+
+* The Metadata IP has to be routed so the bootstrap nodes can fetch the required information
+  to provision the node and the cluster. The Neutron DHCP server also has to be enabled.
+
+  This can be achieved by specifying a route when creating the subnet:
+
+        openstack subnet create --dhcp --host-route destination=169.254.169.254/32,gateway=$ROUTER_IP" (...)
+
+  (where the gateway is the IP of the router in charge of routing the traffic of the Provider Network).
+
+**Note:** This should not apply to OVN because Metadata is completely distributed across OpenStack compute nodes
+          and served from the ovnmeta-xxxxx-xxxx namespace. Therefore, there is nothing to do.
+
+* When deployment baremetal workers on provider networks with OVN, the subnet should be created with DHCP enabled
+  and the Neutron DHCP agent should be deployed and running. There is an [ongoing effort][6] to remove that
```

Comment:
Hmm, the bz points to effort about removing the need for dhcp agent with ovn for BM workers. I'm not sure it applies to our case here. What is providing DHCP on the subnet otherwise?

Reply:
i'll remove that section for now and improve later if needed.